docs/apworld specification.md

```diff
@@ -23,18 +23,61 @@ Metadata about the apworld is defined in an `archipelago.json` file inside the z
 The current format version has at minimum:
 ```json
 {
-    "version": 6,
-    "compatible_version": 5,
+    "version": 7,
+    "compatible_version": 7,
     "game": "Game Name"
 }
 ```
 
-with the following optional version fields using the format `"1.0.0"` to represent major.minor.build:
+The `version` and `compatible_version` fields refer to Archipelago's internal file packaging scheme
+and get automatically added to the `archipelago.json` of an .apworld if it is packaged using the 
+["Build apworlds" launcher component](#build-apworlds-launcher-component),
+which is strongly recommended instead of trying to write these yourself.  
```

Comment:
From skimming the discussions in the thread on discord, it seems like instead of "strongly recommended" it's more like "do not put these in yourself ever", so the wording should be updated to convey that